src/main/scala/beam/agentsim/agents/household/FastHouseholdCAVScheduling.scala

```diff
@@ -430,7 +430,7 @@ object HouseholdTripsHelper {
     var firstPickupOfTheDay: Option[MobilityRequest] = None
     breakable {
       householdPlans.foldLeft(householdNbOfVehicles) { case (counter, plan) =>
-        val usedCarOut = plan.trips.sliding(2).foldLeft(false) { case (usedCar, Seq(prevTrip, curTrip)) =>
+        val usedCarOut = plan.trips.sliding(2).map(slide => (slide.head, slide.last)).foldLeft(false) { case (usedCar, (prevTrip, curTrip)) =>
```

Comment:
This appeases an exhaustiveness warning as error. That said, it will blow up if there is no head or tail...however it already is the case - except it would blow up as a match miss. So, the question is if we want to keep the exceptions or handle the empty and single trip cases?